docs(blog): Introducing Runes - mention .svelte.js suffix (#11522)
sveltejs/svelte: documentation/blog/2023-09-20-runes.md

## Beyond components

> With runes, reactivity extends beyond the boundaries of your `.svelte` files. Suppose we wanted to encapsulate our counter logic in a way that could be reused between components. Today, you would use a [custom store](https://learn.svelte.dev/tutorial/custom-stores) in a `.svelte.js` or `.svelte.ts` file:

Reviewer: A custom store wouldn't be in a `.svelte.js` file - this sentence is talking about how you'd do this in Svelte 4.

Author: Oh yeah good catch 😅

> With runes, things get much simpler:

Reviewer: If we want to mention `.svelte.js` in prose, it should be somewhere around here instead.

Author: How about now?

```

```diff
<!--- file: App.svelte --->
<script>

> 	import { createCounter } from './counter.svelte.js';

Reviewer 2: should this change be called out?

```suggestion
-	import { createCounter } from './counter.js';
+	import { createCounter } from './counter.svelte.js';
```

</button>
```

> > NOTE: Runes can only be used in `.svelte.js` or `.svelte.ts` files.

Reviewer 2: <img width="125" alt="image" src="https://github.com/sveltejs/svelte/assets/1162160/f1c1dc78-e0a4-4374-abd1-6a8127353769">

```suggestion
> Runes can only be used in `.svelte.js` or `.svelte.ts` files.
```

Reviewer: ```suggestion
> Runes can only be used in `.svelte` and `.svelte.js`/`.svelte.ts` files.
```

Alternative suggestion - I think it might make sense to group JS and TS together like this.

</button>
```

> > Runes can only be used in `.svelte.js` and `.svelte.ts` files.

Reviewer: Part of Rich's (and my) suggestion here was that we should also mention `.svelte` files. It's confusing to say runes only work in `.svelte.js` and `.svelte.ts`.

Author: How about the new language? I didn't wanna put `.svelte`, '.svelte.js' and `.svelte.ts` all together, too many svelte's

Author: (It's already implied it works in .svelte files)

> > Runes can only be used in `.svelte.js` and `.svelte.ts` outside of `.svelte` files.

Reviewer 2: how about this?

```suggestion
> Outside `.svelte` components, runes can only be used in `.svelte.js` and `.svelte.ts` modules.
```

Author: Seems like passive speech. I remember we had a discussion last year in `svelte.dev revamp` channel about direct speech vs passive speech, direct won. 😅

I'd go with direct speech preferably.

Reviewer 2: Both are passive voice — `runes can only be used`. Active voice would describe someone _using_ the runes, and that would mean making the actor (you) explicit rather than implicit — instead of 'runes can only be used <by you>', it would become 'you can only use runes'.

Putting the 'outside of `.svelte` files' at the end, rather than in a leading clause, introduces potential ambiguity — it could be the runes that are outside of `.svelte` files, or `.svelte.js` and `.svelte.ts`. The latter is nonsensical, but you have to spend extra time processing the sentence to realise that.

(This is what happens to a person if you spend long enough being yelled at by newsroom sub-editors.)

Author: I forgot ur time at NYT before making that suggestion lol 😅🤣
